divio_cli/localdev/main.py

```diff
@@ -387,10 +391,11 @@ def get_db_restore_command(self):
         return cmd.format(self.db_dump_path)
 
 
-def pull_media(client, stage, remote_id, path=None):
+def pull_media(client, stage, remote_id=None, path=None):
     project_home = utils.get_project_home(path)
     website_id = utils.get_aldryn_project_settings(project_home)['id']
     website_slug = utils.get_aldryn_project_settings(project_home)['slug']
+    remote_id = remote_id or website_id
```

Comment:
Was the the bug fix, and the rest is just adding explicit parameters?

Reply:
Yes indeed!

Reply:
It would be helpful if this line change was in a separate commit, but not mandatory.

Reply:
Agree definitely 👍